tests/unit/mesh/test_pmesh.cpp

```diff
@@ -102,12 +102,14 @@ TEST_CASE("ParMeshGlobalIndices",  "[Parallel], [ParMesh]")
 
             // Verify that the global indices range from 0 to globalN-1.
             {
-               const HYPRE_Int localMin = gi.Min();
-               const HYPRE_Int localMax = gi.Max();
+               const HYPRE_BigInt localMin = gi.Min();
+               const HYPRE_BigInt localMax = gi.Max();
 
                HYPRE_BigInt globalMin, globalMax;
-               MPI_Allreduce(&localMin, &globalMin, 1, HYPRE_MPI_INT, MPI_MIN, MPI_COMM_WORLD);
-               MPI_Allreduce(&localMax, &globalMax, 1, HYPRE_MPI_INT, MPI_MAX, MPI_COMM_WORLD);
+               MPI_Allreduce(&localMin, &globalMin, 1, HYPRE_MPI_BIG_INT, MPI_MIN,
+                             MPI_COMM_WORLD);
+               MPI_Allreduce(&localMax, &globalMax, 1, HYPRE_MPI_BIG_INT, MPI_MAX,
+                             MPI_COMM_WORLD);
```

Comment:
It doesn't seem possible for these to become `HYPRE_BigInt`. The `gi` array is populated by `ParMesh` methods which only return `HYPRE_Int` and these reduce operations are only calling for min or max.

Reply:
Perhaps the `ParMesh::GetGlobal*Indices` methods need to be updated as well.

Reply:
I guess a more appropriate fix here is to just change line 108 to be `HYPRE_Int globalMin, globalMax;`. I was just concerned about the type mismatch from `HYPRE_Int` to `HYPRE_BigInt` in the reduction.

Reply:
Mark, you make a good point: `ParMesh::GetGlobal*Indices` should use `HYPRE_BigInt` type for global indices. For example the code here: https://github.com/mfem/mfem/blob/9a73c2d94c5ecff6dd64c9f7f575e55da0b250cf/mesh/pmesh.cpp#L6062 currently, this converts `HYPRE_BigInt` to `HYPRE_Int`.

Reply:
Yes, this seems to be done in a few places and would cause issues when using HYPRE_MIXEDINT I imagine. I can open a separate PR for this when I get a moment to find all the cases if that sounds like a good plan to you.